Products/ZenUI3/browser/resources/js/zenoss/EventPanels.js

```diff
@@ -1498,6 +1498,11 @@
                     this.applyState(state);
                     this.filterRow.storeSearch();
                 } catch(e) { }
+            // for cases with not encoded parameters
+            } else if (qs) {
+                state = {"filters": Ext.urlDecode(qs)};
+                this.applyState(state);
+                this.filterRow.storeSearch();
```

Comment:
lines 1504 and 1505 are identical to 1498 and 1499. This code can be modified so that the if/else stuff is only for the purpose of getting and validating the `state` variable (and handling exceptions in that validation), then the code flows through to apply the state and update the store.